sdk/communication/azure-communication-chat/README.md

```diff
@@ -240,16 +243,17 @@ def decide_to_retry(error, **kwargs):
     return True
 
 retry = [thread_participant for thread_participant, error in create_chat_thread_result.errors if decide_to_retry(error)]
-chat_thread_client.add_participants(retry)
+if len(retry) > 0:
+    chat_thread_client.add_participants(retry)
```

Comment:
`add_participants` should be a no-op if passed an empty list, so a user should be able to do:
```python
chat_thread_client.add_participants([p for p, error in create_chat_thread_result.errors if decide_to_retry(error)])
```
Additionally, if you need to check that a list has 0 elements, you can just do this in python:
```python
need_to_retry = [p for p, error in create_chat_thread_result.errors if decide_to_retry(error)]
if need_to_retry:
    chat_thread_client.add_participants(need_to_retry)
```

Reply:
`add_participants` has been updated to be no-op if passed empty list. Will update the README and the samples in a separate PR.